runtime/integ.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Any reason this is `runtime/integ.go` instead of `runtime/integ_test.go`?

Reply:
I was trying to add this file as runtime/internal/integ.go first to only expose a few functions, but I couldn't since the runtime package is declared as main. Now I agree that it would be better to rename that to integ_test.go.